src/styles.scss

```diff
@@ -202,7 +202,8 @@ body {
   }
 
   .user-profile {
-    border-radius: 5px;
-    width: 125px;
+    border-radius: 50%;
+    width: 100%;
+    height: 100%;

```

Comment:
it is not required, we will have to either force the ratio of image or crop if we want to keep image circular